### PR TITLE
Frontend libs 13 migration guide

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
Changes:
- added migration guide for FE libs 13
- updated dependencies

### CODE REVIEW

File: website/docs/migrations/12-13.md [hunk 1]
```diff
@@ -0,0 +1,792 @@
...
+- `isObject`
+- `isEqual`
+- `debounce`
+- `throttle`
+
+### Helpers with new names
+
+These helpers should be imported from `@eightshift/ui-components/utilities` instead of Frontend libs:
+- `camelize` &rarr; `camelCase`
+- `classnames` &rarr; `clsx`
```
Comment (iruzevic) [inline website/docs/migrations/12-13.md:58]: we should make these changes for the libs also?

**Review by dadadavorin — DISMISSED** [review]
A huge change indeed, it will be fun times updating this 😅 

But awesome work as always 💪

**Review by iobrado — APPROVED** [review]

File: website/docs/migrations/12-13.md [hunk 2]
```diff
@@ -0,0 +1,792 @@
...
+## Replace components
+
+This will take some time, but the guide below should make it easier. Our recommendation is to go block by block (instead of replacing one type of UI component at a time).
+
+:::note
+Some components will still have fallbacks not to break everything immediately, but replacing them is **strongly** recommended, as everything will be removed in the next version.
+:::
+
+:::tip
+After replacing components, check if there are any new props that might be useful for your usage.
```
Comment (piqusy) [inline website/docs/migrations/12-13.md:259]:
```suggestion
After replacing components, check if there are any new props that might be useful for your situation.
```
Or 
```suggestion
After replacing components, check if there are any new props that might be useful for your project.
```

File: website/docs/migrations/12-13.md [hunk 3]
```diff
@@ -0,0 +1,792 @@
...
+Removed props:
+- `showAsCollapsable`
+
+## Replace Gutenberg components
+
+### `Button`
+
+Replace `Button` from `@wordpress/components`:
+- if `isPressed` is present, with `ToggleButton` from `@eightshift/ui-components`.
+- otherwise, with `Button` from `@eightshift/ui-components`.
```
Comment (piqusy) [inline website/docs/migrations/12-13.md:765]:
```suggestion
- otherwise, replace with `Button` from `@eightshift/ui-components`.
```

**Review by piqusy — DISMISSED** [review]
Nice work 👏🏼 

Left a few suggestions

File: website/docs/migrations/12-13.md (hunk 2 above)
Comment (goranalkovic-infinum) [reply website/docs/migrations/12-13.md:259]: I'll go with _with your use case_, but thanks for the idea 😄

**Review by piqusy — APPROVED** [review]

**Review by iruzevic — APPROVED** [review]

**Pull request merged**